app/templates/components/widgets/forms/image-upload.hbs

```diff
@@ -8,9 +8,20 @@
         <div class="ui text loader">{{t 'Image Uploading'}}</div>
       </div>
     {{/if}}
-    <div class="image">
-      <img src="{{selectedImage}}" alt="Selected image">
-    </div>
+    {{#if isProfileRoute}}
```

Comment:
Wrong way, Remove the logic from widget .

Reply:
Any suggestions?